lib/economic/order.rb

```diff
@@ -62,5 +67,10 @@ def pdf
 
       Base64.decode64(response)
     end
+
+    def register_as_sent
+      response = request(:register_as_sent, "orderHandle" => {"Id" => number})
+      response
```

Comment:
No need to create the intermediary variable here, let's just return the response directly